cmd/operatorcli/main.go

```diff
@@ -16,7 +16,9 @@ var commandEncrypt = &cli.Command{
 	Flags: []cli.Flag{
 		flag.EcdsaPrivateKeyFlag,
 		flag.BlsPrivateKeyFlag,
-		flag.PassphraseFlag,
+		flag.BlsPassphraseFlag,
+		flag.EcdsaPassphraseFlag,
+		flag.EcdsaAliasedPassphraseFlag,
```

Comment:
encrypt command doesn't work on the ecdsa alias key

Reply:
fixed in https://github.com/Eoracle/Eoracle-operator-cli/pull/16/commits/ce047e6cb4ebe44a888bc8e066d0a7ec44786656